lnd.go

```diff
@@ -178,8 +178,10 @@ func lndMain() error {
 	}
 	sCreds := credentials.NewTLS(tlsConf)
 	serverOpts := []grpc.ServerOption{grpc.Creds(sCreds)}
-	grpcEndpoint := fmt.Sprintf("localhost:%d", loadedConfig.RPCPort)
-	restEndpoint := fmt.Sprintf(":%d", loadedConfig.RESTPort)
+	grpcEndpoint := net.JoinHostPort(loadedConfig.RPCHost,
+		strconv.Itoa(loadedConfig.RPCPort))
+	restEndpoint := net.JoinHostPort("",
```

Comment:
The RPC host should also likely be threaded through to allow be the host for the HTTP proxy.